extensions/amp-consent/0.1/test/test-consent-config.js

```diff
@@ -542,6 +546,30 @@ describes.realWin('ConsentConfig', {amp: 1}, (env) => {
 
   describe('expandConsentEndpointUrl', () => {
     it('support expansion in allowed list', async () => {
+      const config = dict({
+        'consentInstanceId': 'test',
+        'checkConsentHref': '/override',
+        'consentRequired': true,
+        'clientConfig': {
+          'test': 'ABC',
+        },
+        'postPromptUI': 'test',
+      });
+      const consentElement = createConsentElement(doc, config);
+      doc.body.appendChild(consentElement);
+      const fakeConsentStateManager = new ConsentStateManager();
+      fakeConsentStateManager.ampdoc_ = doc.body;
```

Comment:
Dont update the private state. Instead do `new ConsentStateManager(doc.body)` above. Update other instances in this file too.